strongbox-cron/strongbox-cron-tasks/pom.xml

```diff
@@ -163,12 +159,10 @@
         </dependency>
 
         <dependency>
-            <groupId>org.apache.directory.server</groupId>
-            <artifactId>apacheds-core</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.directory.server</groupId>
-            <artifactId>apacheds-protocol-ldap</artifactId>
+            <groupId>com.unboundid</groupId>
+            <artifactId>unboundid-ldapsdk</artifactId>
+            <version>5.0.1</version>
```

Comment:
This needs to go into the `strongbox-parent`.

Reply:
fixed